rocket_twin/systems/rocket/rocket_geom.py

```diff
@@ -0,0 +1,43 @@
+from cosapp.base import System
+
+
+class RocketGeom(System):
```

Comment:
This is not a RocketGeom module (that would compute geom dimensions or position). This is more a generic weight and cg evaluation that could be in Dynamic. 

That will simplify the Rocket system.

Reply:
Check